app.js

```diff
@@ -49,7 +49,8 @@ const path = require('path');
 try {
 	require.resolve('sockjs');
 } catch (e) {
-	throw new Error("Dependencies unmet; run npm install");
+	console.error('Dependencies are unmet; run npm install before launching Pokemon Showdown again.');
+	process.exit(1);
```

Comment:
With a ``throw``, you can wrap the module load with ``try-catch``.
``process.exit()`` goes through the native layer and can't be prevented.